keras_trainer/data_generators.py

```diff
@@ -302,6 +325,7 @@ def __init__(self,
                  image_data_generator,
                  iterator_mode=None,
                  random_crop_size=None,
+                 custom_crops=False,
```

Comment:
If this variable contains the crops, should it be initialized to `None` instead? 
```suggestion
                 custom_crops=None,
                 random_crop_size=None,
```

Reply:
This variable is a boolean if True, crops are done. So I would keep the default as `False`.